### DISCUSSION TITLE
Wow

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Chiroyce1/Chiroyce1#4] [General]
https://scratch.mit.edu/projects/635508081/ The best game I've played in a while! Why it only has 14 views? I would follow him.
The only problem is that it's a bit hard

**Comment by Chiroyce1** [comment]
you got me - but LUCKILY i had my volume at 0% 😄

**Reply by Quantum-Codes** [reply]
When will I be able to do that with **Full effect** lol

**Reply by Chiroyce1** [reply]
lol never 😛

**Reply by Quantum-Codes** [reply]
I will get you one day!!

**Comment by Quantum-Codes** [comment]
https://github.com/Quantum-Codes/Scratch-follow-graph we can run python code on github through actions!! 
BTW u want to be tracked by that repo?

**Reply by Chiroyce1** [reply]
nice! sure

**Reply by Quantum-Codes** [reply]
Ok!

**Reply by Quantum-Codes** [reply]
Generated https://github.com/Quantum-Codes/Scratch-follow-graph/blob/main/graphs/Chiroyce.svg

**Reply by Chiroyce1** [reply]
nice

**Comment by Quantum-Codes** [comment]
Requesting Griffpatch's follower history always times out (524)... idk why

**Reply by Chiroyce1** [reply]
i think maybe its too big and it takes a long time to load?

**Reply by Quantum-Codes** [reply]
It fails every single week (since I update every week) . 
I will try re-running and try to request less data maybe (I guess more data might be a problem as u said)

**Reply by Quantum-Codes** [reply]
> Possible special cases include "year" or "month"  ----> in the Scratch DB docs

I think that parameter doesn't work... I think I should put 30 to replace month lol

**Reply by Chiroyce1** [reply]
@Quantum-Codes  why did you fork [this](https://github.com/Quantum-Codes/Chiroyce1) lol? accident?

**Reply by Quantum-Codes** [reply]
Woops. I was trying to make a security thing for private chat :P
Seems like repo owners can only make that

**Reply by Quantum-Codes** [reply]
Also replit not working :(

**Comment by Quantum-Codes** [comment]
There's a color named `indianred` in CSS lol

**Reply by Chiroyce1** [reply]
lol

**Comment by Quantum-Codes** [comment]
Are you getting the @Chiroyce username on github?

**Reply by Chiroyce1** [reply]
No, they said it's not available 😭

**Reply by Quantum-Codes** [reply]
Sad

**Comment by Quantum-Codes** [comment]
Looks like someone is mad at you https://scratch.mit.edu/users/scratch111
Look at his comment on my profile lol

**Reply by Chiroyce1** [reply]
yeah its all a mess.. basically this is what's happening: https://scratch.mit.edu/discuss/post/6204065/

**Reply by Quantum-Codes** [reply]
Exactly. They sleep on appeals too

**Comment by Quantum-Codes** [comment]
Scratch used cleanspeak for filtering right? http://cleanspeak.scratch.mit.edu/ no longer redirects to loginpage. Was there any news about changing of filter services?

_also: any plans on making blogs?_

**Reply by Chiroyce1** [reply]
(1) nope (2) yep, will do after my vacations start.

**Reply by Quantum-Codes** [reply]
Ok thx

**Reply by Chiroyce1** [reply]
also can you go [here](https://github.com/Chiroyce1/ScratchTools) and then press the . key on your keyboard and see where it takes you? it seems to be redirecting me back to the repo itself...

**Reply by Quantum-Codes** [reply]
Um on the repo page? Where exactly?

**Reply by Chiroyce1** [reply]
yeah it should take you back right where you started from

**Reply by Quantum-Codes** [reply]
Nothing happened. I pressed both . Keys (number pad and the one next to shift)

**Comment by Quantum-Codes** [comment]
Za-chary is no longer an ST member?! Since when?

Also I sent an appeal with another email and I got an auto response. I hope they believe that the email is me and not any imposter

**Comment by Quantum-Codes** [comment]
Do you have a discord account or that's an impostor in griffpatich and SA server?

**Reply by Chiroyce1** [reply]
its actually me lol

**Reply by Quantum-Codes** [reply]
Ohhh sorry lol
Can you add me as friend?

**Reply by Chiroyce1** [reply]
sure ig

**Reply by Quantum-Codes** [reply]
Got it
